src/language/__tests__/visitor-test.js

```diff
@@ -17,6 +17,35 @@ import { testSchema } from '../../validation/__tests__/harness';
 import { getNamedType, isCompositeType } from '../../type';
 
 describe('Visitor', () => {
+  it('check path argument', () => {
+    const visited = [];
+
+    const ast = parse('{ a }');
+    visit(ast, {
+      enter(node, key, parent, path) {
+        visited.push(['enter', path.slice()]);
+      },
+
+      leave(node, key, parent, path) {
+        visited.push(['leave', path.slice()]);
+      },
+    });
+
+    console.log(JSON.stringify(visited, null, 2));
```

Comment:
Probably don't want to keep this console.log in the test

Reply:
@mjmahone Consequences of committing at 3 AM 😄 
Removed.